docs/wiki/Home.md

```diff
@@ -0,0 +1,39 @@
+<img src="images/logo_wide_v1.svg" width="250">
+
+The dcrdex wiki provides documentation for dcrdex users and developers. 
+
+Most likely you want to look at the [README](https://github.com/decred/dcrdex/blob/master/README.md) or the [spec](https://github.com/decred/dcrdex/blob/master/spec/README.mediawiki) instead of these wiki pages.
+
+## Contribute changes to the wiki
+
+The following instructions assume that you have already forked **dcrdex**.
+
+1. Initialize your forked wiki on github by navigating to the wiki tab of your forked **dcrdex** repo and clicking the "Create the first page" button.
+
+<img src="images/wiki-creation.png" width="700">
+
+2. In your terminal, navigate to your local **dcrdex** directory and clone the wiki from the **decred/dcrdex** repo.
+
+`git clone https://github.com/decred/dcrdex.wiki.git wiki`
```

Comment:
I guess most of the Home page describing how to contribute to the wiki is incorrect with this change.

Now it's just something like: follow the instructions in docs/wiki/Contribution-Guide.md to clone the dcrdex repository and start a branch for your edits, edit the documents in `docs/wiki`, create a PR with those changes, and once it is merged and administrator will push the changes to the live GitHub Wiki.

Reply:
I've updated the instructions in #1715 what do you think about those?

Reply:
Oh ok so images/wiki-creation.png would still be used.  That makes sense for previewing in your own fork's Wiki.  I have a feeling I'd just do my best to use local previews on my computer's editor for markdown, just not mediawiki.  But maybe I'd get comfy with force pushing the docs/wiki subtree to my wiki remote all the time.

Reply:
I suppose I'd have a remote called `thewiki` for `https://github.com/decred/dcrdex.wiki.git`, and another called `mywiki` for `https://github.com/chappjc/dcrdex.wiki.git` to make sure I don't get them mixed up if I'm making a wiki PR for instance.

Reply:
For someone with write access I guess so... I haven't actually tried yet with `https://github.com/decred/dcrdex.wiki.git` I wonder if I can write there because I'm a contributor? I'm not asking for write permissions, just wondering.

> I have a feeling I'd just do my best to use local previews on my computer's editor for markdown

Your IDE?

Reply:
Anyway, do you want me to change the instructions in this pr? I was just planning on doing the move here but can move the instructions comment here.

Reply:
Yeah, IDE or other markdown enabled editor like ReText.  There's a million of them.

Misses your last question, but I think this PR establishes the baseline docs/wiki folder that exactly matches the wiki at dcrdex.wiki.git, so it's perfect as-is.